bin/convert-argv.js

```diff
@@ -72,15 +72,27 @@ module.exports = function(...args) {
 		const defaultConfigFileNames = ["webpack.config", "webpackfile"].join("|");
 		const webpackConfigFileRegExp = `(${defaultConfigFileNames})(${extensions.join("|")})`;
 		const pathToWebpackConfig = findup(webpackConfigFileRegExp);
-
 		if (pathToWebpackConfig) {
 			const resolvedPath = path.resolve(pathToWebpackConfig);
 			const actualConfigFileName = path.basename(resolvedPath);
 			const ext = actualConfigFileName.replace(new RegExp(defaultConfigFileNames), "");
 			configFiles.push({
 				path: resolvedPath,
-				ext,
+				ext
+			});
+		} else {
+			const pathToWebpackConfig = findup(webpackConfigFileRegExp, {
```

Comment:
If I understand correctly then using `findup` here is unnecessary. 
```
.
├── .webpack
│   └── webpack.config.js
└── packages
    └── foo
        └── .webpack  
            └── webpack.config.js
```
If you have this project structure and you run webpack in `./package/foo` it will resolve the config (`./packages/foo/.webpack/webpack.config.js`).
```
.
├── .webpack
│   └── webpack.config.js
└── packages
    └── foo
```
But if you don't have a webpack config inside foo, then the default config will be used, because it will perform the following checks.
1, `./packages/foo/.webpack` -- no match
2, `./packages/` -- no match
3, `.` -- no match, `findup-sync` will retrun `.webpack` (not it's content) and it won't match our current regex.